corehq/apps/locations/views.py

```diff
@@ -212,6 +212,7 @@ def page_context(self):
             'has_location_types': has_location_types,
             'can_edit_root': self.can_access_all_locations,
             'location_search_help': ExpandedMobileWorkerFilter.location_search_help,
+            'user_is_domain_admin': self.request.couch_user.is_domain_admin(self.domain),
```

Comment:
nit: I'd call this `can_edit_location_fields` to illustrate that this is how that's defined.

Reply:
Going to leave this as `user_is_domain_admin`. I think if it was a composite logic like `A and B`, it would make sense to be more specific about it and name it like that. For now it's just context that if user is domain admin and then it's up to the template to use it as it needs.